src/generic/PermGroups.jl

```diff
@@ -331,19 +362,19 @@ function (G::PermGroup)()
    return z
 end
 
-function (G::PermGroup)(a::Array{Int, 1}; checked=true)
-   length(a) != G.n && error("Unable to coerce to permutation")
+function (G::PermGroup)(a::Array{Int, 1}; checked::Bool=true)
```

Comment:
I literally meant `(a::Array{Int, 1}, check::Bool=true)`, so there won't be a keyword argument. Functions with keyword arguments are slow.